src/main/java/teammates/ui/webapi/action/SearchInstructorsAction.java

```diff
@@ -26,13 +29,40 @@ public void checkSpecificAccessControl() {
         }
     }
 
+    private void addAdditionalSearchFields(InstructorsData instructorsData) {
+        instructorsData.getInstructors()
+                .forEach((InstructorData instructor) -> {
+                    if (instructor.getGoogleId() != null) {
+                        AccountAttributes account = logic.getAccount(instructor.getGoogleId());
+                        if (account != null) {
+                            String institute = StringHelper.isEmpty(account.institute) ? "None" : account.institute;
+                            instructor.setInstitute(institute);
+                        }
+                    }
+                    // Hide information
+                    instructor.hideInformationForSearch();
```

Comment:
Since this method is named `add...`, it's probably not a good idea to be hiding (removing) things here. We could either change this method's name or leave the hiding call in the main `execute()` method.

Reply:
ah trudat